src/components/APICatalog/APICatalog.tsx

```diff
@@ -87,7 +87,7 @@ const APICatalog: React.FC<APICatalogProps> = ({ apisToDisplay, limit }) => {
                           <Chip
                             data-test-id={testIds.apiCardVersion}
                             color="secondary"
-                            label={apiDetails.apiVersion}
+                            label={apiDetails.apiVersion ? apiDetails.apiVersion : t("fallbacks.noVersion")}
```

Comment:
Why not use an or? `apiDetails.apiVersion || t("fallbacks.noVersion")`

Reply:
Fair comment, I changed this but I was supposed to go back on it, and I completely forgot. Thanks for letting me know.